app/models/course/assessment/answer/forum_post_response.rb

```diff
@@ -35,6 +35,17 @@ def compute_post_packs
     end
   end
 
+  def compare_answer(other_answer)
```

Comment:
Is it possible to check that the other answer is also a forum post response answer before comparing?

Reply:
Same for all other answer types.